components/data-display/component-preview.tsx

```diff
@@ -69,6 +70,13 @@ export const ComponentPreview = memo(
         return props
       }, [_containerProps])
 
+      const customConfig = extendConfig({
```

Comment:
@taroj1205 
[nits]
I think this process should be done in useAsync.
If you do that, you won't need to change line 88.

Reply:
You're right, I'll fix it.

Reply:
@taroj1205 
Why did you stop using `extendConfig` ?
(You don't have to use it.)

Reply:
@nakayan5 Thanks, I also think I should use `extendConfig`. I will fix it now.